packages/ui/src/lib/tab/Tab.svelte

```diff
@@ -1,21 +1,20 @@
 <script lang="ts">
-import { router } from 'tinro';
-
+export let routerPath: string;
 export let url: string;
```

Comment:
I think we should extract the route logic from the component, and have only a `selected` parameter for the tab.

Reply:
Ok, updating